src/main/scala/com/codacy/client/bitbucket/v2/service/RepositoryServices.scala

```diff
@@ -10,9 +10,9 @@ class RepositoryServices(client: BitbucketClient) {
    * Gets the list of the user's repositories. Private repositories only appear on this list
    * if the caller is authenticated and is authorized to view the repository.
    */
-  def getRepositories(ownerInfo: OwnerInfo): RequestResponse[Seq[Repository]] = {
+  def getRepositories(ownerInfo: OwnerInfo, pagelen: Int = 100): RequestResponse[Seq[Repository]] = {
     client.executePaginated(
-      Request(s"https://bitbucket.org/api/2.0/repositories/${ownerInfo.value}", classOf[Seq[Repository]])
+      Request(s"https://bitbucket.org/api/2.0/repositories/${ownerInfo.value}?pagelen=$pagelen", classOf[Seq[Repository]])
```

Comment:
I would say that pageLen is an `Option` otherwise you are forcing people to use an option the might want to not use.
did you tested with values <= 0 ?
I think the `?` and following should be trimmed in case of `None`

Reply:
It works with pagelen = 0, with pagelen < 0 gives an error "Invalid pagelen"